config/install/migrate_plus.migration.y_lb_demo_events.yml

```diff
@@ -19,9 +19,9 @@ source:
       summary: "All aboard, seniors! Take a 2-hour tour
         of the city on this casual riverboat picnic
         that includes a boxed lunch."
-      contact_email: sXyimJuV@example.com
-      contact_name: 'John Doe'
-      contact_phone: '+1 (888) 123-4567'
+      contact_email: events@ymcaspringfield.org
```

Comment:
There is no such domain
We need to keep example.org or register it in order not confuse end users or do not create false link if anyone would decide to play fraud and register domain to hijack traffic

Reply:
@podarok should be fixed in the last commit